sdk/storage/azure-storage-blobs/swagger/README.md

```diff
@@ -467,6 +469,54 @@ directive:
       $.ArchiveStatus.description = "For blob storage LRS accounts, valid values are rehydrate-pending-to-hot/rehydrate-pending-to-cool. If the blob is being rehydrated and is not complete then this value indicates that rehydrate is pending and also tells the destination tier.";
 ```
 
+### Striped Blob Support
```

Comment:
Any document for this striped blob?

Reply:
Nah, I prefer not. Striped blob is more of an internal only thing.